x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/utils/LicenseUtils.java

```diff
@@ -15,6 +15,11 @@
 import org.elasticsearch.xpack.core.XPackField;
 
 public final class LicenseUtils {
+
+    public static final String SEARCH_APPLICATION_FEATURE = "search application";
```

Comment:
I apologize for the confusion here, when I mentioned that I liked Ioana's suggestion I thought that you were keeping it as an enum. Moving the enum wholesale made it a bit better in my eyes since we weren't creating a new class for the enum. Strings are OK too, but we weren't saying you had to do both. I'll leave that up to you. 

With that said I really dislike the name `Feature` here however it's used. I thought we'd talked about other names like `Product`?

Reply:
No worries. I changed it to `enum` now.